src/test/librbd/test_internal.cc

```diff
@@ -693,7 +693,7 @@ TEST_F(TestInternal, SnapshotCopyup)
   ASSERT_EQ(0, snap_create(*ictx2, "snap1"));
   ASSERT_EQ(0, snap_create(*ictx2, "snap2"));
 
-  ASSERT_EQ(256, api::Io<>::write(*ictx2, 256, bl.length(), bufferlist{bl},
+  ASSERT_EQ(256, api::Io<>::write(*ictx2, 4096, bl.length(), bufferlist{bl},
```

Comment:
The expected return value also needs to be adjusted.  Currently this fails as follows:
```
[ RUN      ] TestInternal.SnapshotCopyup
../src/test/librbd/test_internal.cc:696: Failure
Expected equality of these values:
  256
  api::Io<>::write(*ictx2, 4096, bl.length(), bufferlist{bl}, 0)
    Which is: 4096
[  FAILED  ] TestInternal.SnapshotCopyup (122 ms)
```

Reply:
Corrected, sorry for this inconvenience, I thought this was too simple to rerun the test.